presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java

```diff
@@ -108,7 +109,7 @@ public void testMultiplePositions(double maxStandardError)
         }
 
         assertLessThan(stats.getMean(), 1.0e-2);
-        assertLessThan(Math.abs(stats.getStandardDeviation() - maxStandardError), 1.0e-2);
+        assertLessThan(abs(stats.getStandardDeviation()), 1.0e-2 + maxStandardError);
```

Comment:
nit: We can just test the standard deviation will be no more than `maxStandardError`, so:

```
assertLessThan(stats.getStandardDeviation() - maxStandardError, 1.0e-2);
```

I would also explain the motivation in commit message. 
```
Minor fix to standard error check in approx_distinct test

For types with small value sets (e.g. TINYINT), standard deviation 
can be small or even zero. This will fail the current check since it 
asserts the standard deviation is within a range of maxStandardError.
```